README.md

```diff
@@ -31,7 +31,7 @@ Pipeline frameworks & libraries
 * [Dockerflow](https://github.com/googlegenomics/dockerflow) - Workflow runner that uses Dataflow to run a series of tasks in Docker.
 * [Doit](http://pydoit.org/) - Task management & automation tool.
 * [Drake](https://github.com/Factual/drake) - Robust DSL akin to Make, implemented in Clojure.
-* [Drake (R package)](https://github.com/wlandau-lilly/drake) - R-focused reproducibility and scalable high-performance computing with an easy interface.
+* [drake R package](https://github.com/ropensci/drake) - Reproducibility and high-performance computing with an easy R-focused interface.
```

Comment:
For consistency, can `drake` be started with an uppercase?

Reply:
Yes, done. I also added a note to disambiguate the R package from [Factual's Drake](https://github.com/factual/drake), which is capitalized.